Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Dialogs/UX/Internal/InternalExtendedKryptonForm.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Krypton.Toolkit.Suite.Extended.Dialogs
+{
+    /// <summary>DO NOT USE!!!!!!!!! The purpouse of this is to expand the capabilities of a normal <seealso cref="KryptonForm"/>.</summary>
+    public class InternalExtendedKryptonForm : KryptonForm
```

Comment:
Make it internal, then set the class dll's that extend it to be "Friends" of this dll.